pyproject.toml

```diff
@@ -60,17 +60,32 @@ extend-exclude = """
 )
 """
 
-[tool.isort]
-profile = "black"
-force_single_line = "True"
-known_first_party = ["keras", "tests"]
-default_section = "THIRDPARTY"
-line_length = 80
-extend_skip_glob=[
-    "examples/*",
-    "guides/*",
+[tool.ruff]
+line-length = 80
+
+[tool.ruff.lint]
+select = [
+    "E",  # pycodestyle error
+    "F",  # Pyflakes
+    "I",  # isort
+]
+ignore = [
+    "E722",  # do not use bare 'except'
+    "E721",  # use `is` and `is not` for type comparisons
```

Comment:
Do we really need to ignore this one?

Reply:
Sure, I'll add another commit in this PR to resolve it.